react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const CustomButton = <button>Click Me!</button>;
```

Comment:
This should be written as a function:

```JSX
function CustomButton() {
  return <button>Click Me!</button>;
}
```

Reply:
Oh oops I had been going through this one wondering if I did something wrong because it looked just like the previous one